tests/src/Interop/common/ComHelpers.h

```diff
@@ -17,7 +17,7 @@
 namespace Internal
 {
     template<typename C, typename I>
-    HRESULT __QueryInterfaceImpl(
+    typename std::enable_if<std::is_assignable<I, C>::value, HRESULT>::type __QueryInterfaceImpl(
```

Comment:
Why is this needed?

Reply:
I thought it would be useful, but after thinking over it again, the compiler will likely already do what this was checking for me.

Reply:
Well basically if `I` isn't assignable from `C` this entire QI implementation would be wrong. It has to be true. Yes, please remove.